decider-bcl2fastq/src/main/java/ca/on/oicr/pde/deciders/data/BasesMask.java

```diff
@@ -0,0 +1,277 @@
+package ca.on.oicr.pde.deciders.data;
+
+import ca.on.oicr.pde.deciders.exceptions.InvalidBasesMaskException;
+import java.util.Objects;
+import java.util.logging.Level;
+import java.util.logging.Logger;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.logging.Logger'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)